### PR TITLE
Remove yarn and added ci file

### PR DESCRIPTION
**Pull request by FrankIglesias**
## Summary 
- Removed yarn and changed for npm
- Created ci file
- Completed readme instructions

## Trello cards
https://trello.com/c/3O9UPxeS/13-completar-readme
https://trello.com/c/pfWivnm3/21-integracion-con-circle-ci
https://trello.com/c/MK8I5UQ2/15-agregar-configuraci%C3%B3n-de-repositorio-para-el-kickoff-del-proyecto

### CODE REVIEW

**Review by damfinkel — APPROVED** [review]

**Review by wfolini — APPROVED** [review]

File: generators/app/templates/circle.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+machine:
+ node:
```
Comment (mvbattan) [inline generators/app/templates/circle.yml]: Doesn't this need 2 spaces for indentation ? I always have mistaken this point 😅

File: run.sh [hunk 2]
```diff
@@ -21,23 +21,19 @@ elif ! system_has node; then
...
 fi
 
-
-yarn global add create-react-app --prefix /usr/local > /dev/null 2>&1
+npm i -g create-react-app  > /dev/null 2>&1
 PROJECT_NAME=`echo "${PWD##*/}"`
 cd ..
-create-react-app "${PROJECT_NAME}"
-yarn global add yo generator-react-bootstrap > /dev/null 2>&1
+create-react-app "${PROJECT_NAME}" --use-npm
```
Comment (mvbattan) [inline run.sh:39]: https://www.youtube.com/watch?v=MfXVG8o41RQ

File: run.sh [hunk 3]
```diff
@@ -21,23 +21,19 @@ elif ! system_has node; then
...
+npm i -g yo generator-react-bootstrap > /dev/null 2>&1
 cd $PROJECT_NAME
+echo "Project created"
 yo react-bootstrap --force
-yarn global add package-json-merge > /dev/null 2>&1
+npm i -g package-json-merge > /dev/null 2>&1
```
Comment (mvbattan) [inline run.sh]: I'd extract dependencies within a variable:
```
DEPENDENCIES=(
  yo
  generator-react-bootstrap
  package-json-merge
)

npm i -g ${DEPENDENCIES[*]} > /dev/null 2>&1
```

File: run.sh [hunk 4]
```diff
@@ -21,23 +21,19 @@ elif ! system_has node; then
...
-yarn global add create-react-app --prefix /usr/local > /dev/null 2>&1
+npm i -g create-react-app  > /dev/null 2>&1
 PROJECT_NAME=`echo "${PWD##*/}"`
 cd ..
-create-react-app "${PROJECT_NAME}"
-yarn global add yo generator-react-bootstrap > /dev/null 2>&1
+create-react-app "${PROJECT_NAME}" --use-npm
+npm i -g yo generator-react-bootstrap > /dev/null 2>&1
 cd $PROJECT_NAME
+echo "Project created"
```
Comment (mvbattan) [inline run.sh]:
```
echo "Project $PROJECT_NAME created"
```
?

File: run.sh [hunk 5]
```diff
@@ -21,23 +21,19 @@ elif ! system_has node; then
...
 yo react-bootstrap --force
-yarn global add package-json-merge > /dev/null 2>&1
+npm i -g package-json-merge > /dev/null 2>&1
 package-json-merge package.json packageB.json > packageA.json
```
Comment (mvbattan) [inline run.sh]: This line is creepy, but is the best we can do at moment I think

Comment (FrankIglesias) [reply run.sh]: Merging both packages?... Yes I do agree, however we can create an array of minimum dependencies and do a yarn add of all of them below

**Review by mvbattan — APPROVED** [review]
Seems good ! ✨

**Pull request merged**